e2e/tests/login.spec.ts

```diff
@@ -0,0 +1,114 @@
+import { test, expect } from '@playwright/test'
+
+test.describe('Login successful test', () => {
+  test('Verify user is able to login with admin credentials', async ({
+    page
+  }) => {
+    await page.goto('/RCO')
+
+    const usernameInputField = page.locator('#username')
+    const passwordInputField = page.locator('#password')
+    const loginButton = page.locator('button[type="submit"]')
+    const invalidUserOrPasswordMessage = page.locator(
+      '//div[contains(@class, "MuiSnackbarContent-message")]'
+    )
+
+    await usernameInputField.type('ian')
```

Comment:
Is this still a throwback to the POM strategy?

The above can be represented through this, can't it?
```
await page.locator('#username').type('ian')
```

I _think_ the above strategy would reduce about 20 lines from the file.  Yes, this is very much down to taste, and you have the experience, but I'd like to start with the smallest/simplest solution - and refactor to introduce complexity when it is required.